src/cloudWatchLogs/registry/logDataRegistry.ts

```diff
@@ -80,12 +80,7 @@ export class LogDataRegistry {
         if (isHead && logData.previous?.token === undefined) {
             const msgId = uriToKey(uri)
             // show something so the user doesn't think nothing happened.
-            await Message.putMessage(
-                msgId,
-                `Loading data from log group: '${logData.logGroupInfo.groupName}'`,
-                undefined,
-                500
-            )
+            await Messages.putMessage(msgId, `Loading Log Group: '${logData.logGroupInfo.groupName}'`, undefined, 500)
```

Comment:
Could say "Fetching" or "Searching". Mainly need to avoid a long prefix message so that the message is more readable and the log group name is more visible.